docker-compose.yml

```diff
@@ -23,6 +23,9 @@ x-environment:
     LAGOON_ROUTE: &default-url ${LOCALDEV_URL:-http://govcms8-saas.docker.amazee.io}
     DEV_MODE: ${DEV_MODE:-false}
     X_FRAME_OPTIONS: ${X_FRAME_OPTIONS:-SameOrigin}
+    # Allow to override docker host used from the inside of the containers.
+    DOCKERHOST: ${DOCKERHOST:-}
```

Comment:
# Temporary fix for xdebug.remote_host being incorrectly discovered.
# @see https://github.com/amazeeio/lagoon/issues/1088

On some systems, the upstream xdebug remote host discovery mechanism does not have enough time to get the correct host's hostname, leading to incorrect value of ` xdebug.remote_host` to be wrong and IDE not being able to connect to xdebug session. This line adds an ability to provide `DOCKERHOST` explicitly in `.env` file.